peps/pep-0683.rst

```diff
@@ -2,7 +2,7 @@ PEP: 683
 Title: Immortal Objects, Using a Fixed Refcount
 Author: Eric Snow <ericsnowcurrently@gmail.com>, Eddie Elizondo <eduardo.elizondorueda@gmail.com>
 Discussions-To: https://discuss.python.org/t/18183
-Status: Accepted
+Status: Final
 Type: Standards Track
 Content-Type: text/x-rst
```

Comment:
We don't need this any more:
```suggestion
```

And please could you also delete the redundant emacs stuff from the end?

```rst
..
    Local Variables:
    mode: indented-text
    indent-tabs-mode: nil
    sentence-end-double-space: t
    fill-column: 70
    coding: utf-8
    End:
```